parm/soca/mom6/diag_table

```diff
@@ -0,0 +1,58 @@
+20210322.06Z.C48.64bit.non-mono
+2021 03 22 06 0 0
+
+"fv3_history",    0,  "hours",  1,  "hours",  "time"
+"fv3_history2d",  0,  "hours",  1,  "hours",  "time"
+######################
+"ocn_da%4yr%2mo%2dy%2hr",      0,  "hours", 1, "hours", "time", 1, "hours", "1901 1 1 0 0 0"
+##############################################
+# static fields
+  "ocean_model", "geolon",      "geolon",      "ocn_da%4yr%2mo%2dy%2hr", "all", .false., "none", 2
+  "ocean_model", "geolat",      "geolat",      "ocn_da%4yr%2mo%2dy%2hr", "all", .false., "none", 2
+# ocean output TSUV and others
+ "ocean_model", "SSH",       "ave_ssh",      "ocn_da%4yr%2mo%2dy%2hr","all",.true.,"none",2
+ "ocean_model", "MLD_0125",  "MLD",      "ocn_da%4yr%2mo%2dy%2hr","all",.false.,"none",2
+#===============================================
+
+
+#===============================================
+ "ocean_model","u","u"            ,"ocn_da%4yr%2mo%2dy%2hr","all",.false.,"none",2
+ "ocean_model","v","v"            ,"ocn_da%4yr%2mo%2dy%2hr","all",.false.,"none",2
+ "ocean_model","salt","Salt"      ,"ocn_da%4yr%2mo%2dy%2hr","all",.false.,"none",2
+ "ocean_model","temp","Temp"      ,"ocn_da%4yr%2mo%2dy%2hr","all",.false.,"none",2
+ "ocean_model","h","h"            ,"ocn_da%4yr%2mo%2dy%2hr","all",.false.,"none",2
+
+###
+#=============================================================================================
```

Comment:
Whatever is below will be in the diag_table that we will modify in the ufs config, I would say that needs to go as well.

Reply:
The table was updated (PR2)